sdk/lib/3rdparty/libxml2/config.h

```diff
@@ -138,56 +90,30 @@
 /* Have shl_load based dso */
 /* #undef HAVE_SHLLOAD */
 
-/* Define to 1 if you have the `signal' function. */
-#define HAVE_SIGNAL 1
-
-/* Define to 1 if you have the <signal.h> header file. */
-#define HAVE_SIGNAL_H 1
-
 /* Define to 1 if you have the `snprintf' function. */
 #define HAVE_SNPRINTF 1
 
-/* Define to 1 if you have the `sprintf' function. */
-#define HAVE_SPRINTF 1
-
-/* Define to 1 if you have the `srand' function. */
-#define HAVE_SRAND 1
-
-/* Define to 1 if you have the `sscanf' function. */
-#define HAVE_SSCANF 1
-
 /* Define to 1 if you have the `stat' function. */
 #define HAVE_STAT 1
 
-/* Define to 1 if you have the <stdarg.h> header file. */
-#define HAVE_STDARG_H 1
-
 /* Define to 1 if you have the <stdint.h> header file. */
 #define HAVE_STDINT_H 1
 
+/* Define to 1 if you have the <stdio.h> header file. */
+#undef HAVE_STDIO_H
```

Comment:
This seems weird?

Reply:
Yeah it does... it's not present in `config.h.cmake.in`, which is checked into their git repository. I'm basing this off `config.h.in`, which is shipped in the release archives, and indeed this line was added there between 2.10.1 and 2.10.2.
Guess I should inquire where that file even comes from.

Reply:
Found it... `config.h.in` is generated by autoheader, and the autoconf version used was updates with 2.10.2. `HAVE_STDIO_H` was indeed added, by https://git.savannah.gnu.org/gitweb/?p=autoconf.git;a=commitdiff;h=b045574cb2ea1c8441be7a7b56fd471d704ebf3a
So this is correct and intentional.

Thanks for catching it, though, this is exactly the kind of stuff I'm looking for!

Reply:
But we have `stdio.h`, so shouldn't this be defined to `1` ?

Reply:
Yes, it should be. Good catch!